data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```
@@ -151,5 +151,26 @@
     "calories": 152,
     "description": "Dehydrated chocolate milk powder.  Mix with water to make drinkable milk.",
     "fun": -1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "licorice",
+    "name": "licorice",
+    "symbol": "%",
+    "volume": "40 ml",
+    "weight": "78 g",
+    "spoils_in": "230 days",
+    "color": "red",
+    "comestible_type": "FOOD",
+    "material": [ "junk" ],
+    "description": "licorice, a swirly confectionary treat in strings of sweet tasting candy.",
```

```suggestion
    "description": "A swirly confectionary treat in strings of sweet tasting candy.",
```
Also probably should be placed into core game files, rather than mod files.